views/openacademy_menu.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- ACTIONS OPENING VIEWS ON MODELS -->
+    <record id="action_window_course" model="ir.actions.act_window">
+      <field name="name">openacademy.menu_course</field>
+      <field name="res_model">openacademy.course</field>
+      <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="action_window_session" model="ir.actions.act_window">
+      <field name="name">openacademy.menu_session</field>
+      <field name="res_model">openacademy.session</field>
+      <field name="view_mode">tree,form</field>
+    </record>
+
+    <!-- TOP MENU ITEM -->
+    <menuitem id="openacademy.menu_root" name="Open Academy"/>
```

Comment:
When the external ID refers to the same module, it's not required to specify the module name as it's assumed:

```suggestion
    <menuitem id="menu_root" name="Open Academy"/>
```

This also applies for the below menus